### ISSUE TITLE
Graphite+Grafana Dashboard

### ISSUE THREAD

**Issue by mddunn** [confluentinc/jmx-monitoring-stacks#37]
Customer is interested in an example Graphite+Grafana implementation as they love the Prom+Grafana one but are using Graphite and want to replicate.

**Comment by jeqo** [comment]
Haven't see that much usage of Graphite on the field myself.
Is there a supported way to export JMX to a Graphite backend? Checking this one https://github.com/logzio/jmx2graphite, it piggybacks on Jolokia.

Let us know if there's still interest on this one.

**Comment by jeqo** [comment]
@mddunn Thanks for the links! totally forgot about telegraf 😅.

Would you be interested on contributing the initial pipeline/dashboards for this stack? I'd be happy to help/review it.

**Comment by hifly81** [comment]
@mddunn we have recently added a stack with jolokia2,telegraf and influxdb. I suppose we should only replace influx with graphite to have this done. wdyt?

**Comment by mddunn** [comment]
@hifly81 I think that would be helpful, but I remember this customer was also interested in true Grafana+Graphite demo. Might be worth having both?

**Comment by hifly81** [comment]
> @hifly81 I think that would be helpful, but I remember this customer was also interested in true Grafana+Graphite demo. Might be worth having both?

jolokia/telegraf/graphite/grafana?

**Comment by mddunn** [comment]
Yeah that seems like a valid setup. I think jmxtrans is another popular tool that people use to get JMX metrics into Graphite
